Azure/iotedge
# Add a by-value comparer for service accounts...

...so we don't update service accounts unless they actually change.

To test in k8s,  create a deployment and:
1. Watch the secrets in the namespace
2. Watch the serviceaccount of a module.

You should only see one service account token for each module.  When the system is stable, the service account for the module should not change, and should only change when the module is added or removed.
